pkg/virt-launcher/virtwrap/statsconv/util/domstats_utils.go

```diff
@@ -157,8 +157,8 @@ var Testdataexpected = `{
      "Time": 86393420788, 
      "TimeSet": true, 
      "User": 1620000000, 
-     "UserSet": true
-   }, 
+     "UserSet": true,
```

Comment:
I think you should emit the `,` from the last parameter, as it was before. Perhaps this is the reason the unit tests are failing.

Reply:
done